int32 package_id
string package_name
string chute_name
string dock_station_name

---

string result